bundles/org.openhab.binding.pilight/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pilight"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Pilight Bridge</label>
+		<description>Pilight Bridge which connects to a Pilight instance.</description>
+
+		<properties>
+			<property name="firmwareVersion">-</property>
+		</properties>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Network Address</label>
+				<description>The IP or host name of the Pilight instance.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="integer" required="true" min="1" max="65335">
+				<label>Port</label>
+				<description>Port of the Pilight daemon. You must explicitly configure the port in the Pilight daemon config or
+					otherwise a random port will be used and the binding will not be able to connect.
+				</description>
+				<default>5000</default>
+			</parameter>
+			<parameter name="delay" type="integer" required="false" min="1" max="65335">
+				<label>Delay between commands</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
				<label>Delay between Commands</label>
```